express-notes-api/server.js

```diff
@@ -0,0 +1,122 @@
+import express from 'express';
+import { writeFile, readFile } from 'node:fs/promises';
+const app = express();
+const port = 8080;
+
+async function getFiles() {
+  try {
+    const filePath = new URL('data.json', import.meta.url);
+    const content = await readFile(filePath, 'utf8');
+    return content;
+  } catch (error) {
+    console.error('Error');
+  }
+}
+
+async function processFiles(url, content) {
+  try {
+    await writeFile('data.json', content);
+    console.log(content);
+  } catch (error) {
+    console.error('Error');
+  }
+}
```

Comment:
If reading to or writing from files fails during a request, the server should send a `500 Internal Server Error` response.